CoEpi/ble/Central.swift

```diff
@@ -4,22 +4,14 @@ import Foundation
 import CoreBluetooth
 import os.log
 import UIKit
-import RxRelay
 
-struct DetectedPeripheral {
-    let uuid: UUID
-    let name: String?
+protocol CentralDelegate: class {
```

Comment:
See my other comments, main problem here is that this will (most likely) be replaces with a covidwatch lib and I doubt that they want to force the users to use RX. It's something that can be discussed, but it's ok to just use delegates/callbacks for now in this part of the app.

Reply:
That's fair. I'd throw in an argument for callbacks over delegates, easier to manage closures than delegate objects IMO, but I suppose the difference is minor